### ISSUE TITLE
grid checkbox reversed

### ISSUE THREAD

**Issue by hrgdavor** [microsoft/maker.js#515]
current meker.js website

![image](https://user-images.githubusercontent.com/2480762/148921837-edcd2717-339a-488f-b202-82275a7a6ca3.png)

![image](https://user-images.githubusercontent.com/2480762/148921866-83de3e0f-6291-4fdd-8572-12c74de25571.png)

**Comment by danmarshall** [comment]
I've seen this also, sometimes. Not sure, but it seems the browser is trying to be smart about caching the value??

**Comment by hrgdavor** [comment]
You are right, it could be that if code is not checking checkbox value but depending on internal state,  and browsers to tend to try to restore input value sometimes on refresh. ... it is probably something weird like that...

**Comment by hrgdavor** [comment]
ok, found a way to reproduce. Tested in Chrome and Brave (just in case maybe is there difference) and firefox
steps added to main text aboeve
